script/test.ps1

```diff
@@ -0,0 +1,7 @@
+$env:XDG_CONFIG_HOME = "$PWD/.test-config"
+$env:XDG_STATE_HOME = "$PWD/.test-state"
```

Comment:
@Olical I think this line should get added to the other test script so that the trust db is isolated from the user's config